Flappy Bird Clone/Assets/Scripts/Player/PlayerBehaviour.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Input;
+using ScriptableObjects;
+using UnityEngine;
+
+namespace Player
+{
+    public class PlayerBehaviour : MonoBehaviour
+    {
+        [SerializeField] private PlayerInfo playerInfo;
+
+        private InputController _inputController;
+
+        private SpriteRenderer _spriteRenderer;
+        private Rigidbody2D _physicsComponent;
+
+        private void Awake()
+        {
+            _inputController = FindFirstObjectByType<InputController>();
```

Comment:
Out of curiosity: Is this dynamically created? Is this could be assigned in editor instead being assigned on the fly?

Reply:
Yeah, it definitely could! Gonna change it right now!